File: Problemas/282A - Bit++/282A_Bit++.py
# 282A - Bit++
# https://codeforces.com/problemset/problem/282/A

# Inputs:
# 1) Número de statements
statements = int(input())

# Variable X
x = 0

for indice_statement in range(0, statements):
    
    # Inputs:
    # 2) Statement (X++, X--, ++X, --X)
    line = input()

    # Si el statement tiene ++, suma 1 a x
    # Si el statement tiene --, resta 1 a x
    if('++' in line):
        x += 1
    if('--' in line):
        x -= 1

# Imprime el valor final de x
print(x)

File: Problemas/282A - Bit++/282A_Bit++.cpp
// 282A - Bit++
// https://codeforces.com/problemset/problem/282/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
using namespace std;

int main() {

    // Inputs:
    // 1) statements = Número de statements
    int statements; 
    cin >> statements;

    // Variable X    
    int x = 0;           

    // Lee cada statement
    for(int indice_statement = 0; indice_statement < statements; indice_statement++) {

        // Inputs:
        // 2) operacion = Statement (X++, X--, ++X, --X)
        string operacion; 
        cin >> operacion;

        // Si el statement tiene en medio un +, entonces suma 1 a x
        // Si el statement tiene en medio un -, entonces resta 1 a x
        if(operacion[1] == '+') {
            x++;
        } else if(operacion[1] == '-') {
            x--;
        }
    }

    // Imprime el valor final de X
    cout << x << endl;

    return 0;
}
